backend/open_webui/retrieval/vector/utils.py

```
@@ -0,0 +1,14 @@
+from datetime import datetime
+
+
+def stringify_metadata(
+    metadata: dict[str, any],
+) -> dict[str, any]:
+    for key, value in metadata.items():
+        if (
+            isinstance(value, datetime)
+            or isinstance(value, list)
+            or isinstance(value, dict)
```

Ruff (0.12.2)

Multiple `isinstance` calls for `value`, merge into a single call

Merge `isinstance` calls for `value`

(SIM101)